### PR TITLE
Add URL for Supporters section in home page

### PR DESCRIPTION
**Pull request by coder12git**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Issue -> #415

* **Other information**:
In .docusaurus/ routes file I have added this code -
{
    path: "/supporters",
    component: Supporters, 
    exact: true,
 },

### CODE REVIEW

**Comment by coder12git** [comment]
> * **Please check if the PR fulfills these requirements**
> 
> * [x]  The commit message follows our guidelines
> * [x]  Tests for the changes have been added (for bug fixes / features)
> * [x]  Docs have been added / updated (for bug fixes / features)
> 
> * **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
>   Issue -> [Add url `https://kubeedge.io/supporters` for Supporters section in home page #415](https://github.com/kubeedge/website/issues/415)
> * **Other information**:
>   In .docusaurus/ routes file I have added this code -
>   {
>   path: "/supporters",
>   component: Supporters,
>   exact: true,
>   },


https://github.com/kubeedge/website/assets/108334168/aad4242a-3320-43a2-9f74-e666cbc597cf

**Comment by fisherxu** [comment]
@coder12git Thanks for the contributions, but I think it should jumper to the supporters section instead of jumper a new page..

**Comment by coder12git** [comment]
> @coder12git Thanks for the contributions, but I think it should jumper to the supporters section instead of jumper a new page..

Okay, I will do that!

**Comment by coder12git** [comment]
@fisherxu Please review now, done all the changes!

https://github.com/kubeedge/website/assets/108334168/821f961f-c9fc-4e2b-928e-bf891b6a89b5

File: docusaurus.config.js [hunk 1]
```diff
@@ -155,6 +155,11 @@ const config = {
             position: "left",
             target: "_self",
           },
+          {
+            label: "Supporters",
+            href: "/#supporters",
+            position: "left",
+          },
```
Comment (fisherxu) [inline docusaurus.config.js:162]:
```suggestion
```
No need to add to nav bar  :)

Comment (coder12git) [reply docusaurus.config.js]: > 

Ok, I thought it might be helpful, anyway I removed that!

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**